src/d/parser/expression.d

```diff
@@ -1043,3 +1046,80 @@ StringLiteral parseStringLiteral(ref TokenRange trange) {
 	
 	return new StringLiteral(location, name.toString(trange.context));
 }
+
+/**
+ * Parse floating point literals
+ */
+FloatLiteral parseFloatLiteral(ref TokenRange trange) {
+	const location = trange.front.location;
+	auto litString = trange.front.toString(trange.context);
+
+	trange.match(TokenType.FloatLiteral);
+	import d.common.builtintype : BuiltinType;
+
+
+	assert(litString.length > 1);
+	//Look for a suffix
```

Comment:
```
// space before text
```

This is more important than you'd expect, because when doing this type of things, you signal lack of attention to details. In other terms, you are priming the reviewer to expect your code is garbage. This is generally not what you want to be doing.

Reply:
I was under the impression we were going to sdfmt this file sooner rather than later

Reply:
Yes we are. But it won't reformat comments, but also, this is not about the comment, this is about what you communicate to me.

I know you, so I know you are a good dev. I see that from anyone, I have to assume the code is trash. Not paying attention to this will deserve you.